lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -79,7 +79,7 @@ def _create_node_capabilities(
         node_capabilities.memory_mb = 4096
         node_capabilities.disk = schema.DiskOptionSettings()
         node_capabilities.network_interface = schema.NetworkInterfaceOptionSettings()
-        node_capabilities.network_interface.max_nic_count = 1
+        node_capabilities.network_interface.max_nic_count = 0
```

Comment:
I missed the initial PR so likely missing some context. Does 0 max sense here?
Unrelated to this PR, I wonder if we want to specify number of cpu(s) in the node capability?

Reply:
It should be like below,
```python
node_capabilities.network_interface.nic_count = 1
node_capabilities.network_interface.max_nic_count = 1
```